src/helpers/rerun-pr-workflows.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { context as githubContext } from '@actions/github';
+import { octokit } from '../octokit';
+
+export class RerunPrWorkflows {
+  branch = '';
+  //maybe fork name should be an optional input?
+  // forkName?: string;
+}
+
+export const rerunPrWorkflows = async ({ branch }: RerunPrWorkflows) => {
+  // list all workflow runs
+  const allRuns = await octokit.request('GET /repos/{owner}/{repo}/actions/{branch}', {
```

Comment:
didn't know you could do this haha. i think to be in line with the other helpers we may want to use [this method](https://octokit.github.io/rest.js/v18#actions-list-workflow-runs) which accomplishes the same thing

Reply:
I don't know if my method actually works, because it is failing on that step when I try to run the unit tests. I'll refactor to use the other method and see how that goes.

Reply:
(as long as this still gives us the same data)

Reply:
Yeah let's be consistent and use the [octokit](https://github.com/ExpediaGroup/github-helpers/blob/main/src/octokit.ts#L18) object for github API calls